styles/styles.css

```
@@ -94,49 +168,115 @@ body {
 /* end slider */
 
 /* start banner */
-
+.banner {
+  display: block;
+  margin-bottom: 81px;
+}
 .banner-header{
+  width: 434px;
+  margin: auto auto;
```

```suggestion
  margin: auto;
```
И еще в двух местах где так